ActiveLabel/ActiveLabel.swift

```diff
@@ -327,7 +327,9 @@ typealias ElementTuple = (range: NSRange, element: ActiveElement, type: ActiveTy
     // MARK: - Auto layout
 
     open override var intrinsicContentSize: CGSize {
-        return super.intrinsicContentSize
+        textContainer.size = CGSize(width: self.preferredMaxLayoutWidth, height: CGFloat.greatestFiniteMagnitude)
+        let size = layoutManager.usedRect(for: textContainer)
+        return CGSize(width: ceil(size.width), height: ceil(size.height))
```

Comment:
Why `ceil`? Also if you don't care about `width` calculations you can just return `super. intrinsicContentSize.width ` here, but check if this value differs from your `size`.